File: Trigonometric-functions/Java/Trigonometricfunctions.java
public class Trigonometricfunctions {
    public static void main(String[] args) {
        final int iterations = 100_000_000;
        System.out.printf("計算回数: %,d%n", iterations);

        double result = 0.0;
        for (int i = 0; i < iterations; i++) {
            result += Math.sin(i) * Math.cos(i);
        }

        System.out.println("計算結果: " + result);
    }
}

File: Trigonometric-functions/C#/Trigonometricfunctions.cs
using System;

class Program
{
    static void Main()
    {
        const long iterations = 100000000;
        Console.WriteLine($"計算回数: {iterations:N0}");

        double result = 0.0;
        for (long i = 0; i < iterations; i++)
        {
            result += Math.Sin(i) * Math.Cos(i);
        }

        Console.WriteLine($"計算結果: {result}");
    }
}